etc/webapi.xml

```diff
@@ -73,9 +73,15 @@
         </resources>
     </route>
     <route url="/V1/extend/integration/pp/upsert" method="POST">
-        <service class="Extend\Integration\Api\ProductProtectionInterface" method="upsert" />
+        <service class="Extend\Integration\Api\ProductProtectionInterface" method="upsertSession" />
         <resources>
             <resource ref="anonymous" />
         </resources>
     </route>
+    <route url="/V1/extend/integration/pp/upsert/:cart_id" method="POST">
+        <service class="Extend\Integration\Api\ProductProtectionInterface" method="upsertCartId" />
+        <resources>
+            <resource ref="Extend_Integration::manage" />
```

Comment:
Setting this as protected for now as the main use case will be Cypress

Reply:
I understand the goal but what does`ref="Extend_Integration::manage"` mean from a technical standpoint?

Reply:
In regards to the ACL we've added: https://github.com/helloextend/magento-extend-integration/blob/c060fd7153adff75e1a9d163cb600e853e8b6a4a/etc/acl.xml#L13

So our integration's access token has that resource included meaning it can be used to hit that endpoint: https://github.com/helloextend/magento-extend-integration/blob/c060fd7153adff75e1a9d163cb600e853e8b6a4a/etc/integration/api.xml#L14